public/index.php

```diff
@@ -19,6 +19,8 @@
     Debug::enable();
 }
 
+$_SERVER['HTTP_X_FORWARDED_FOR'] = $_SERVER['HTTP_X_REAL_IP'];
```

Comment:
What if it will be not set and `HTTP_X_FORWARDED_FOR` will have real value? As it can be populated by nginx config rules? I would add some checks, but best would be to handle this on nginx level - not here.